src/org/zaproxy/zap/extension/pscanrulesBeta/resources/help/contents/pscanbeta.html

```diff
@@ -12,6 +12,22 @@
 
 <H2>Charset Mismatch</H2>
 This check identifies responses where the HTTP Content-Type header declares a charset different from the charset defined by the body of the HTML or XML. When there's a charset mismatch between the HTTP header and content body Web browsers can be forced into an undesirable content-sniffing mode to determine the content's correct character set.
+This check handles various conditions depending on the Threshold set for the scanner, as follows:
+<ul>
+  <li>All Thresholds:</li>
```

Comment:
> that is, closing "li" tag should be after the closing "ul" tag
Huh? How can I close the line item after closing the un-ordered list?

Or do you just mean that you want the starts and ends of the lists on the same line as the first and last items of the lists?